gpbft/gpbft.go

```diff
@@ -20,38 +20,38 @@ import (
 type Phase uint8
 
 const (
-	INITIAL_PHASE Phase = iota
-	QUALITY_PHASE
-	CONVERGE_PHASE
-	PREPARE_PHASE
-	COMMIT_PHASE
-	DECIDE_PHASE
-	TERMINATED_PHASE
+	InitialPhase Phase = iota
+	QualityPhase
+	ConvergePhase
+	PreparePhase
+	CommitPhase
+	DecidePhase
+	TerminatedPhase
```

Comment:
Is there any syntax or construct we can use in Go that will give permission to use ALL_CAPS for these constants? For these ones in particular I think their use is much clearer in all caps.

Reply:
We could keep those as all caps if that makes it clear. I have no strong opinions.